static/src/stylesheets/module/commercial/_advert.scss

```diff
@@ -110,7 +110,7 @@
     }
 }
 
-.advert__kicker {
+.advert__kicker__slash {
```

Comment:
I'm in two minds about whether to give this class a new name or not.. It would make removing it later on much simpler, but at the same time if we've missed any uses of the `advert__kicker` class then this will break the styling for those.

It looks like there's only one hit for `advert__kicker` in Frontend templates. But -- though I could be wrong about this -- I believe it _sometimes_ happens that markup from elsewhere (e.g. thrashers) will implicitly depend on global styles which are inserted by Frontend. That said, the only other occurrences of `advert__kicker` that I can find in the guardian org are in [commercial-templates/legacy](https://github.com/guardian/commercial-templates/blob/d7ebcab920dd9b3ab55a1a000019faa1cb81f2e7/legacy/src/capi-multiple-paidfor/app/app-capi-multiple.js#L34) so maybe we don't need to worry too much..

Any thoughts?

[gh search results for class name](https://github.com/search?q=org%3Aguardian%20advert__kicker&type=code)

Reply:
I had a similar thought here but given the limited scope of this class it seems to be ok especially as we keep the class name with the line break. That said, I'm not wedded to it so happy to change if you think its worth it.